test/avrora/encoder_test.exs

```diff
@@ -706,6 +753,10 @@ defmodule Avrora.EncoderTest do
       119, 32, 97, 114, 101, 32, 121, 111, 117, 63, 0>>
   end
 
+  defp numeric_transfer_plain_message_0 do
```

Comment:
Maybe to emphasize the issue we can name it `numeric_transfer_plain_message_with_fake_magic_byte`

Reply:
done